nameko/events.py

```diff
@@ -183,10 +168,69 @@ def __init__(self, source_service, event_type, handler_type=SERVICE_POOL,
         super(EventHandler, self).__init__(
             queue=None, requeue_on_error=requeue_on_error)
 
+    @property
```

Comment:
I find myself using a `property` when the alternative method would be `get_foo()` with no arguments.